tests/factory/test_factory_unused_options.py

```diff
@@ -1,72 +1,90 @@
 import copy
 import os
 
-import mozfile
 import pytest
 
 from mozdownload.utils import urljoin
 from mozdownload import FactoryScraper
 
 
-@pytest.mark.parametrize('scraper_type,builds,fname,arg', [
+@pytest.mark.parametrize('filename,args', [
```

Comment:
Same here in regards of `args`, and the changed number of arguments.

Reply:
https://github.com/mozilla/mozdownload/pull/537#discussion_r262687689 - So should i keep it same as `filename, args` ?

Reply:
See the other comment.